README.rst

```diff
@@ -1,23 +1,178 @@
-Open edX Webhooks
-=================
+Open edX Webhooks Handlers (and Other JIRA/GitHub Utilities)
+============================================================
 
 Webhooks for `Open edX`_ integrating `JIRA`_ and `Github`_. Designed to
-be deployed onto `Heroku`_.
+be deployed at `Heroku`_.
+
+`Access the app`_ at https://openedx-webhooks.herokuapp.com.
 
 |Build Status| |Coverage Status| |Documentation badge|
 
-How to Run Tests
-----------------
+Set Up Development Environment
+------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Make sure you've installed:
+
+-  Python 2.7.x development environment (virtualenv strongly
+   recommended)
+
+   *Hint*: See `runtime.txt`_ for the exact version
+-  `Heroku Command Line`_
+
+   All ``heroku`` commands can be performed through the Heroku web-based
+   dashboard as well, if you don't want to use the CLI.
+
+Set up
+~~~~~~
+
+1. Log in using the email address and password you used when creating
+   your Heroku account:
+
+   .. code:: sh
+
+       heroku login
+
+   Authenticating is required to allow both the ``heroku`` and ``git``
+   commands to operate.
+
+2. Add the Heroku app repo as a git remote:
+
+   .. code:: sh
+
+       heroku git:remote -a openedx-webhooks-staging
+
+3. Verify that the remote is added properly:
+
+   .. code:: sh
+
+       git remote -v
+
+   You should see output similar to:
+
+   .. code:: sh
+
+       heroku  https://git.heroku.com/openedx-webhooks-staging.git (push)
+       heroku  https://git.heroku.com/openedx-webhooks-staging.git (fetch)
+       origin  git@github.com:edx/openedx-webhooks.git (fetch)
+       origin  git@github.com:edx/openedx-webhooks.git (push)
+
+Develop
+-------
+
+This app relies on the following addons from Heroku:
+
+-  PostgreSQL
+-  Redis
+-  IronMQ
+-  Papertrail
+-  Scheduler
+
+While it's possible to replicate the entire stack locally, it'll be
+difficult to ensure consistent experience for each developer. Instead,
+we utilize the `pipeline`_ facility offered by Heroku to handle our
+development needs.
+
+The general development cycle is:
+
+Code → Deploy branch to staging → Test → Iterate
+
+To deploy a local branch to staging:
+
+.. code:: sh
+
+    git push heroku [branch_or_tag_or_hash:]master
```

Comment:
My point is that there's a literal command that will always be what you want, rather than having a placeholder here that needs to be filled in with the name of the current branch.